### ISSUE TITLE
Suggestion: Brew that reduces Alcohol %

### ISSUE THREAD

**Issue by TKwSni** [DieReicheErethons/Brewery#309]
Basically add a function that let us make recipes that reduce Alcohol %

I tried making a recipe with negative alcohol but it seems is not possible.

**Comment by mcgodbuild** [comment]
Have you considered making a potion and then set that potion at the start of the config where milk and bread are placed as items that take drunkiness away. If thats not waht your suggesting then sorry

**Comment by AaronPB** [comment]
It does not seem to be working. Could you give an example @mcgodbuild ?
I've tried adding it by name and id:
```
- Brewery: beer_example/5
- Brewery: Cerveza Ahumada/5
- Brewery: '&6Cerveza &f&oAhumada'/5 (throws null exception)
```
Also without "Brewery" but it does not work either.

**Comment by mcgodbuild** [comment]
Which part does not work? making the potions or adding them as items that take drunkness away?

**Comment by AaronPB** [comment]
Adding it to the drainItems section. The recipe works fine.

**Comment by mcgodbuild** [comment]
can you paste the entire config file in a browser viewer like https://hastebin.com/ so we can view everything and perhaps come up with a solution?

**Comment by mcgodbuild** [comment]
Also can you add vanilla items to the drain items without any issues?

**Comment by AaronPB** [comment]
Here you are: https://pastebin.pl/view/208be129 (it is a default config with the recipe, still not draining)
Vanilla items are working well, I checked with an Apple item and it drains correctly.

**Comment by mcgodbuild** [comment]
Did you try to use its actual name? 

recipes:
   beer_example:
      name: '&eCerveza Ahumada &8&oInsípida/&eCerveza Ahumada &7&oEquilibrada/&eCerveza Ahumada &f&oNatural'

**Comment by AaronPB** [comment]
Sure, but it didn't work.
I have tried with:
```
- Brewery:&eCerveza Ahumada &f&oNatural/20
- Brewery:'&eCerveza Ahumada &f&oNatural'/20
- Brewery:'&eCerveza Ahumada &8&oInsípida/&eCerveza Ahumada &7&oEquilibrada/&eCerveza Ahumada &f&oNatural'/20
```

**Comment by AaronPB** [comment]
Well, thanks for your help @mcgodbuild but stil not working...
Maybe @Sn0wStorm could verify if it is actually possible?
If you could give me an example that works, I will appreciate it.

**Comment by Sn0wStorm** [comment]
I don't think this works, at least i never tested or implemented it. The only way i know of is to use the vanilla drain items

**Comment by Sn0wStorm** [comment]
Possible since 3.1
